js/deck.js

```diff
@@ -0,0 +1,51 @@
+class Deck {
+    constructor(size) {
+        this.size = size;
+        this.cards = []
+    }
+        newDeck() {
+            const names = ['2', '3', '4', '5', '6', '7', '8', '9', '10', 'J', 'Q', 'K', 'A'];
+            const suits = ['Hearts','Diamonds','Spades','Clubs'];
+            // es gribēju kko šādu lai vairāk par vienu kavu varētu būt bet man neiet
+            // for( let i = 0; i < Deck.size; i++ ) {
+            //     for( let j = 0; j < suits.length; j++ ) {
+            //         for( let n = 0; n < names.length; n++ ) {
+            //             this.cards.push(new Card(n+2+(52*i), names[n], suits[j]))
+            //     }
+            // }
+            // }
+                for( let j = 0; j < suits.length; j++ ) {
```

Comment:
Indentācija.

Reply:
Jā, definējam cards.json failu, kuru ielasīsim iekš index.html, tad šeit mums būs pieejams šis json objekts un iesim vienkārši tam cauri un viss kods reducēsies uz:
```js
for(let i = 0; i < cards.length; i++ ) {
  const { name, suite, values } = cards[i]
  this.cards.push(new Card(name, suite, values))
}
```